bot/exts/moderation/dm_relay.py

```diff
@@ -90,7 +90,11 @@ async def on_message(self, message: discord.Message) -> None:
         # Handle any attachments
         if message.attachments:
             try:
-                await send_attachments(message, self.webhook)
+                await send_attachments(
+                    message,
+                    self.webhook,
+                    username=f"{message.author.display_name} ({message.author.id})"
```

Comment:
You must use `sub_clyde` for the username.

Reply:
> You're correct that antispam sends the attachments before deleting the messages. However, setting `use_cached` to false decreases the likelihood of successfully sending deleted attachments for all callers of the function. This is because a message can be deleted by the author while the bot is trying to process/handle the message. That being said, I don't know if it's really worth potentially making two requests per attachment via the idea to try the proxy and then the normal if the proxy fails.

I see, I don't think it would be really be worth trying with `use_cached` and then without, as without should work the vast majority of the time. Trying without `use_cached` and with it if that fails would probably be best if we thought deleted messages could be a problem, but I'm not sure it's that important.

Reply:
It depends on the kind of accuracy the mod team wants, but yes, I tend to agree that it isn't worth using. Anyway, you still need to fix the sub_clyde issue. Now that I think about it, it'd be more convenient to do it within `send_attachments` i.e. 

```py
    webhook_send_kwargs = {
        'username': message.author.display_name,
        'avatar_url': message.author.avatar_url,
    }
    webhook_send_kwargs.update(kwargs)
    webhook_send_kwargs['username'] = sub_clyde(webhook_send_kwargs['username'])
```